lib/hexpm/web/templates/docs/faq.html.md

```diff
@@ -1,8 +1,70 @@
 ## FAQ
 
-### How should I name my packages?
+### Packages
+
+#### Why is my package documentation returning "page not found"?
+
+This could be for a number of reason. When you run `mix hex.publish [docs]` the documentation is
+generated on your machine and then uploaded to Hex. Issues with documentation usually stem from
+the documentation generation. First verify that your documentation works locally by running
+`mix docs`, also verify that there are no differences in letter casing since Hexdocs is
```

Comment:
running `mix docs` and then opening the generated files locally, typically `doc/index.html`.